src/app/components/Curation/index.tsx

```diff
@@ -91,6 +95,8 @@ export default ({
       );
     case RADIO_SCHEDULE:
       return <RadioSchedule initialData={radioSchedule} />;
+    case EMBED:
+      return vjEmbed ? <Embed oembed={embed} type="" /> : null;
```

Comment:
Think we could probably just return `<Embed oembed={embed} type="" />` rather than having a ternary. The `Embed` component itself should return `null` if there is no html for it to render.